docs/deployments/deploy-chrome-extension.mdx

```diff
@@ -15,30 +15,33 @@ For Clerk production instances, there must be a domain associated with the insta
 
 <Steps>
 
-### Copy your production Clerk key and frontend API Url
+### Set your Clerk API keys
```

Comment:
This should probably keep the `Frontend API URL` as the user is copying one key and then one URL, versus keys like with most frameworks.